api/src/modules/indicator-records/indicator-records.service.ts

```diff
@@ -342,10 +342,9 @@ export class IndicatorRecordsService extends AppBaseService<
   }
 
   /**
-   * @description Creates Indicator-Records for each Sourcing-Record, by first retrieving Raw Indicator data from the DB, then applying
+   * @description Creates Indicator-Records for a single Sourcing-Record, by first retrieving Raw Indicator data from the DB, then applying
```

Comment:
This change is true for the time being, but it will change once we handle:

https://vizzuality.atlassian.net/browse/LANDGRIF-625

Worth changing it now or after said tasks is tackled?